examples/plot_mne_example.py

```diff
@@ -0,0 +1,223 @@
+# FOOOF models across the power spectra. A list of FOOOFGroup objects is returned.
+#
+
+###################################################################################################
+
+# Initialize a FOOOF group object, with desired settings
+fg = FOOOFGroup(peak_width_limits=[1, 6], min_peak_amplitude=0.075,
+                    max_n_peaks=6, peak_threshold=1, verbose=False)
+
+###################################################################################################
+
+# Shaping the PSDs so that we can run fooofgroup
+fooof_psd = np.squeeze(psd[4,:,:]) 
+n_channels, n_freq = fooof_psd.shape
+num_blocks = len(mne.read_events(event_fname))
+
+# Setting frequency range
+freq_range = [3, 32]
+
+# This returns a list of FOOOFGRoup objects
+fg.fit(freqs, fooof_psd, freq_range)
+fg.plot()
+
+###################################################################################################
+
+# Define bands of interest
+bands = {'theta': [2,7],
+         'alpha': [8,14],
+         'beta': [15,30]}
+
+# Creating dictionaries to store all the aperiodic properties across frequencies 
+results = {}
+for band_name in bands.keys():
+    results[band_name] = np.zeros(shape=[ num_blocks, n_channels, n_feats])
+
+# Creating dictionaries to store all the periodic properties across frequencies 
+slope_results = np.zeros(shape=[num_blocks, n_channels, 2])
+
+###################################################################################################
+
+# Populating periodic and aperiodic values 
+for block in range(0, num_blocks):
+    for ind, res in enumerate(fg):
+        slope_results[block, ind, :] = res.background_params
+        for band_label, band_range in bands.items():
+            results[band_label][block, ind,  :] = get_band_peak(res.peak_params, band_range, True)
+
+###################################################################################################
+# Plotting topologies 
+# --------------
+# 
+# The following will illustrate the process of exploring the previoulsy generated 
+# oscillatory values 
+
+###################################################################################################
+# This is the function required to create color bars for topographies 
+def plot_topo_colorbar(vmin, vmax, label):
+    """
+    Create a colorbar for the topography plots
+    vmin: int
+    vmax: int
+    label: str
+    """
+    fig = plt.figure(figsize=(2, 3))
+    ax1 = fig.add_axes([0.9, 0.25, 0.15, 0.9])
+
+    cmap = cm.viridis
+    norm = colors.Normalize(vmin=vmin, vmax=vmax)
+
+    cb1 = colorbar.ColorbarBase(plt.gca(), cmap=cmap,
+                                norm=norm, orientation='vertical')all_fg.plot()
+
+###################################################################################################
+
+# Settings: In this example, we will be plotting the Alpha Central Frequency
+# and oscillatory slope 
+band = 'alpha'
+cur_data = results[band]
+feats = ["CFS", "AMPS", "BWS"]
```

Comment:
Terminology: we've also moved to talking about 'power' instead of 'amplitude' (since power is the real measurement), so I would update this this 'PWS' instead of 'AMPS'.